documentation/design-docs/dotnet-tools.md

```diff
@@ -398,6 +399,49 @@ CONVERT
       Writing:       ./trace.speedscope.json
       Conversion complete
 
+REPORT
+
+    dotnet-trace report [-h|--help]
+                        [-t|--type]
+                        [-p|--parameters]
+                        [-o|--output <output_file_path>]
+                        --header
+                        <trace_file>
+
+    Generate report into stdout from a previously generated trace_file.
+
+    -h, --help
+        Show command line help
+    
+    -t, --type
+        Specifies the type of report to create. The first type of report that will be available is 'topN'
+
+    -p, --parameters
+        Takes a list of arguments for each specific report type, see below for details.
+        type            parameters  explanation
+        'topN'          -n          the number of top functions to gather on the stack
+                        -i          find the top n methods inclusively on the stack. Without this tag, will find the top N methods exclusively on the stack
+        'countEvent'    
```

Comment:
@josalem If the latest commit looks a bit redundant, it is because there are two different options to display and explain the parameters for each report type. The first option is on lines 414-419. The second option is on lines 428-443.